lib/util.js

```diff
@@ -211,28 +211,28 @@ function debuglog(set) {
  * @param {Object} obj The object to print out.
  * @param {Object} opts Optional options object that alters the output.
  */
-/* legacy: obj, showHidden, depth, colors*/
-function inspect(obj, opts) {
+function inspect(obj, opts, /* legacy */depth, /* legacy */colors) {
```

Comment:
Well, I do not have a strong opinion.
I went ahead and changed it back to a similar way as before to make you happy ;-)